monty/dev.py

```diff
@@ -29,8 +41,61 @@ def deprecated(replacement=None, message=None, category=FutureWarning):
         Original function, but with a warning to use the updated class.
     """
 
-    def craft_message(old, replacement, message):
+    def _convert_date(date: tuple[int, int, int]) -> datetime:
```

Comment:
don't think we need a helper function for this. this can be inlined

Reply:
Sorry. This looks like an oversight from my side.